dev/com.ibm.ws.logging_fat/test-applications/RealFlushTestApp/src/com/ibm/ws/logging/flush/fat/printObjectTests/DummyObject.java

```diff
@@ -0,0 +1,49 @@
+package com.ibm.ws.logging.flush.fat.printObjectTests;
+
+public class DummyObject {
+	
+	public String small;
```

Comment:
need to use indentation consistent with rest of OpenLiberty project (eclipse setting...)